src/main/java/com/ericsson/eiffel/remrem/publish/service/SendResult.java

```diff
@@ -1,14 +1,22 @@
 package com.ericsson.eiffel.remrem.publish.service;
 
```

Comment:
If it is used by the framework to generate the correct structure of the output we should keep it. Perhpas PublishResult? Then you need to rename the existing PublishResult to PublishResultItem.